src/tctrack/tempest_extremes/tempest_extremes.py

```diff
@@ -133,6 +133,49 @@ class TEOutputCommand(TypedDict):
     """
 
 
+class TEThreshold(TypedDict):
+    """Data required for a threshold filter for a track in StitchNodes.
+
+    Any tracks that do not satisfy the threshold value for a given number of points will
+    be filtered out. Each condition is of the form "var,op,value,count" and conditions
+    are separated by ";".
+
+    See Also
+    --------
+    TETracker.stitch_nodes : The StitchNodes call from the TETracker object.
+    StitchNodesParameters : The StitchNodes parameter class.
+
+    References
+    ----------
+    TempestExtremes Documentation: https://climate.ucdavis.edu/tempestextremes.php#StitchNodes
+    Source: https://github.com/ClimateGlobalChange/tempestextremes/blob/master/src/nodes/StitchNodes.cpp
+
+    Examples
+    --------
+    To add a filter requiring latitude ``"lat"`` to be less than 40 degrees for
+    10 or more points in each track:
+
+    >>> TEOutputCommand(var="lat", op="<=", value=40, count=10)
+    {"var": "lat", "op": "<=", "value": 40, "count": 10},
+    """
+
+    var: str
+    """Name of the variable being tested. "col" in TempestExtremes."""
```

Comment:
Maybe clarify this maps to the col option in Tempest Extremes. I initially read it as "col" being the name of the variable that will be tested.